Example/IntegrationTests/Pairing/PairingTests.swift

```diff
@@ -70,7 +70,9 @@ final class PairingTests: XCTestCase {
         let pushLogger = ConsoleLogger(suffix: prefix + " [Push]", loggingLevel: .debug)
         walletPairingClient = pairingClient
         let echoClient = EchoClientFactory.create(projectId: "", clientId: "", echoHost: "echo.walletconnect.com", environment: .sandbox)
-        walletPushClient = WalletPushClientFactory.create(logger: pushLogger,
+        let keyserverURL = URL(string: "https://keys.walletconnect.com")!
```

Comment:
In Chat SDK I'm injecting keyserverURL as string. URL object is better you think?

Reply:
it's probably safer?